style/haskell/sample.hs

```diff
@@ -1,3 +1,6 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes #-}
+
```

Comment:
I don't know that we need a rule about this, but I *don't* prefer this extra whitespace.